modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/fragment/renderer/FDSViewFragmentRenderer.java

```diff
@@ -260,6 +260,20 @@ private String _buildFragmentHTML(
 			fragmentEntryLink.getCompanyId(), fdsEntryObjectEntryERC,
 			fdsEntryObjectDefinition);
 
+		Collection<ObjectEntry> fdsCardObjectEntries = null;
```

Comment:
It should be `fdsCardsSection*` and `fdsListSection*` throughout this PR, to match naming in persistence.